### ISSUE TITLE
HotModuleReplacementPlugin is added twice with webpack-dev-server >= 3.2.0

### ISSUE THREAD

**Issue by steinm91** [webpack-contrib/grunt-webpack#172]
### Webpack/Grunt Configuration (webpack.config.js, webpack part of Gruntfile.js)

```json
{
  "webpack": { 
    "options": {
      "hot": "true",
      "inline": "true"
    },
    "dev": "..." 
  }
}
```

### Expected Behavior

When activating HMR, it should either update the changed module or execute a page reload. 

### Current Behavior

Currently the HotModuleReplacementPlugin is added twice to the list of webpack plugins.
This leads to the error: _RangeError: Maximum call stack size exceeded_ and neither the hot module gets replaced nor the page reload is executed.

This happens due to the fact that webpack-dev-server adds the HotModuleReplacementPlugin since version 3.2.0 if HMR is enabled, see https://github.com/webpack/webpack-dev-server/commit/178e6cc3f604750620e9de557517e583924334a8?diff=unified.

### Possible Solution

The task "webpack-dev-server" of grunt-webpack calls _addDevServerEntrypoints_ with the provided options, which then adds HotModuleReplacementPlugin to the plugin array. Immediately afterwards "webpack-dev-server" adds the module again since there is no check if the module is already present.
(see https://github.com/webpack-contrib/grunt-webpack/blob/master/tasks/webpack-dev-server.js#L117)

I suggest to simply remove this code which adds the plugin, since _addDevServerEntrypoints_ already does it.
If this is not possible or desired one can also check if the module is already in the array before adding it again.

### Your Environment

| software         | version
| ---------------- | -------
| webpack          | 4.29.5
| grunt-webpack    | 3.1.3
| grunt            | 1.0.3
| node             | >=8
| npm              | >=5
| Operating System |

**Comment by ArmorDarks** [comment]
Confirming the same issue. Did someone figure out a workaround?

**Comment by evilebottnawi** [comment]
Feel free to send a PR

**Comment by danez** [comment]
Fixed in 4.0.0
